rel/overlay/etc/default.ini

```diff
@@ -68,6 +68,9 @@ default_engine = couch
 ; inadvertently abusing partitions resulting in hot shards. The default
 ; is 10GiB. A value of 0 or less will disable partition size checks.
 ;max_partition_size = 10737418240
+;
+; Start node in single_node mode so default databases are created immediately.
+;single_node = true
```

Comment:
@janl this should be `;single_node = false` because the default is false. I'll file a PR